SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -92,6 +92,25 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 		});
 	}
 
+	FEditorDelegates::PostPIEStarted.AddLambda([this](bool bIsSimulatingInEditor)
+	{
+		if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking() && GIsAutomationTesting)
+		{
+			LocalDeploymentManager->IsServiceRunningAndInCorrectDirectory();
+			LocalDeploymentManager->GetLocalDeploymentStatus();
```

Comment:
Why this? If it's required before starting for some reason shouldn't it be in `VerifyAndStartDeployment` or further down the stack?

Reply:
This is an interesting one, we do these two calls on a timer every 3 seconds (? maybe 3 times a second), and it is possible when running tests that the deployment status is cached incorrectly and starting a local deployment will fail, which is why I added these checks here explicitly.